### PR TITLE
Doc change for PR#4188.

### PR DESCRIPTION
**Pull request by evanchaoli**
This is the doc change for concurse/concourse#4188.

Signed-off-by: Chao Li <chaol@vmware.com>

### CODE REVIEW

File: lit/docs/jobs.lit [hunk 1]
```diff
@@ -67,18 +67,29 @@ configured job consists of the following fields:
...
 
+  \define-attribute{minimum_succeeded_builds: number}{
+    \italic{Optional.} Keep at lease specified succeeded builds. For example,
+    if specifies \reference{job-build-log-retention-builds} to 5, and this
+    attribute to 1, say a job has a build history: 7(f), 6(f), 5(f), 4(f),
+    3(f), 2(f), 1(s), where f means failed and s means succeeded, then builds 2
+    and 3 will be reaped, because it retains 5 build logs, and at lease 1
+    succeeded build log. Default is 0.
+  }{job-build-log-retention-minimum-succeeded-builds}
+
```
Comment (deniseyu) [inline lit/docs/jobs.lit]: I wasn't involved at all with the original PR (which is probably a good thing now - fresh eyes and all  👀 ) so this paragraph reads a bit confusing to me! Going to suggest a multi-line rewrite, hopefully this works...

File: lit/docs/jobs.lit [hunk 2]
```diff
@@ -67,18 +67,29 @@ configured job consists of the following fields:
     \italic{Optional.} Keep logs for the last specified number of builds.
   }{job-build-log-retention-builds}
 
+  \define-attribute{minimum_succeeded_builds: number}{
+    \italic{Optional.} Keep at lease specified succeeded builds. For example,
```
Comment (deniseyu) [inline lit/docs/jobs.lit]:
```suggestion
    \italic{Optional.} Keep a minimum number of successful build logs that would normally be reaped. 
    Requires \reference{job-build-log-retention-builds} to be set to an integer higher than 0 in order to work.
```

File: lit/docs/jobs.lit [hunk 3]
```diff
@@ -67,18 +67,29 @@ configured job consists of the following fields:
...
+    if specifies \reference{job-build-log-retention-builds} to 5, and this
```
Comment (deniseyu) [inline lit/docs/jobs.lit]:
```suggestion
    For example, if \reference{job-build-log-retention-builds} is set to 5, and this
```

File: lit/docs/jobs.lit [hunk 4]
```diff
@@ -67,18 +67,29 @@ configured job consists of the following fields:
...
+    attribute to 1, say a job has a build history: 7(f), 6(f), 5(f), 4(f),
```
Comment (deniseyu) [inline lit/docs/jobs.lit]:
```suggestion
    attribute to 1, say a job has the following build history: 
    7(f), 6(f), 5(f), 4(f), 3(f), 2(f), 1(s),
```

File: lit/docs/jobs.lit [hunk 5]
```diff
@@ -67,18 +67,29 @@ configured job consists of the following fields:
...
+    3(f), 2(f), 1(s), where f means failed and s means succeeded, then builds 2
```
Comment (deniseyu) [inline lit/docs/jobs.lit]:
```suggestion
    where f means failed and s means succeeded, then builds 2
```

File: lit/docs/jobs.lit (hunk 4 above)
Comment (deniseyu) [reply lit/docs/jobs.lit]: (btw I know this line break won't render this way in the docs, but it makes it easier to reason about in Markdown)

File: lit/docs/jobs.lit [hunk 6]
```diff
@@ -67,18 +67,29 @@ configured job consists of the following fields:
...
+    and 3 will be reaped, because it retains 5 build logs, and at lease 1
```
Comment (deniseyu) [inline lit/docs/jobs.lit]:
```suggestion
    and 3 will be reaped, because it retains 5 build logs, and at least 1
```

**Comment by evanchaoli** [comment]
@deniseyu Thanks for polishing the wording. Please help review again. This feature has been marked with release 5.7.

**Pull request merged**